### PR TITLE
Refine e2e tests

### PR DESCRIPTION
**Pull request by STRRL**
### What problem does this PR solve?

Fix https://github.com/chaos-mesh/chaos-mesh/issues/1136;

### What is changed and how does it work?
This PR will do things like:
- split each testcases from `test/e2e/chaos/basic.go` into different files.
- appending test step description with `ginkgo.By()`

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by STRRL** [comment]
@YangKeao @Yisaer @cwen0 , PTAL

**Review by YangKeao — APPROVED** [review]
LGTM

File: test/e2e/chaos/iochaos/io_delay.go [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package iochaos
+
+import (
+	"context"
+	"github.com/chaos-mesh/chaos-mesh/api/v1alpha1"
+	"github.com/chaos-mesh/chaos-mesh/test/e2e/util"
```
[the remark this answers is not in the document]
Comment (STRRL) [reply test/e2e/chaos/iochaos/io_delay.go]: fixed, PTAL

**Review by cwen0 — APPROVED** [review]
LGTM

**Review by YangKeao — APPROVED** [review]
LGTM

**Pull request merged**